hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardFsck.java

```diff
@@ -599,6 +613,12 @@ private void checkForViolationInPairs(Path file,
   private void checkNoViolationInPairs(Path file2,
       List<S3GuardFsck.ComparePair> comparePairs,
       S3GuardFsck.Violation violation) {
+
```

Comment:
It would be nice if you can add some comments here. Thanks.

Reply:
I don't see what you mean by that - the logging is sufficient here. Do you mean to add javadoc to the method?

Reply:
Yes I meant javadoc only.

Reply:
done